hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -2126,6 +2127,92 @@ private class Emit(
           }
         }
 
+      case x@NDArrayConcat(nds, axis) =>
+        val inputType = coerce[PArray](nds.pType2)
+        val inputNDType = coerce[PNDArray](inputType.elementType)
+
+        val ndType = coerce[PNDArray](x.pType2)
+        val codeNDs = emit(nds, env, er, None)
+
+        val inputArray = mb.newField[Long]
+        val n = mb.newField[Int]
+        val i = mb.newField[Int]
+
+        val loadAndValidateArray = Code(
+          inputArray := codeNDs.value[Long],
+          n := inputType.loadLength(inputArray),
+          (n < 1).orEmpty(Code._fatal("need at least 1 array to concatenate")))
```

Comment:
Should we say `ndarray` here? Numpy goes back and forth between saying ndarray and array, but they don't have a separate list like array type the way we do.

Reply:
A few lines below you use `NDArray` in your message, let's be consistent between the two